sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -68,10 +68,6 @@ OoyalaSkinBridgeListener.prototype.onTimeChange = function(e) { // todo: naming
     cuePoints: e.cuePoints
   });
 
-  if (e.caption) {
-    this.skin.setState({caption: e.caption});
```

Comment:
now both vod cc and live cc go through onClosedCaptions event. it no longer a part of time changed event. This makes implementation more error proof.